overwatch-rs/src/services/relay.rs

```diff
@@ -130,6 +134,10 @@ impl<M> OutboundRelay<M> {
             .blocking_send(message)
             .map_err(|e| (RelayError::Send, e.0))
     }
+
+    pub fn into_sink(self) -> impl Sink<M> + Send + 'static {
+        PollSender::new(self.sender)
+    }
```

Comment:
This was way easier than actually implement `Sink` for the `OutboundRelay` itself.